### PR TITLE
fix(compose): Make layout more responsive

### PR DESCRIPTION
**Pull request by kvviecien**
Better responsiveness and more organized layout for mobile

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -856,7 +843,7 @@ compose input.mat-chip-input {
 }
 
 compose .mat-standard-chip {
-    bottom: 6px !important;
+    // bottom: 6px !important;
```
Comment (tadzik) [inline src/styles.scss:846]: I think this commented-out bit can go now :)

Comment (kvviecien) [reply src/styles.scss:846]: Right, I was probably checking something out and forgot about it as always.

**Comment by kvviecien** [comment]
In my opinion, the Send button being bigger looks silly right next to the other ones being different size. Like someone made a mistake styling it.
If it's necessary to make it stand out, I'd rather make it a different color. Maybe grey out the other ones, take off a little bit of their saturation, and leave the Send button like it is now.

**Comment by kvviecien** [comment]
Reordered the buttons and moved HTML and Attach buttons closer to the textarea.

I noticed the attachments looked a little wonky on mobile like this \/ so I fixed them too!
Screenshot 2020-06-25 at 23 08 06

Result:
Screenshot 2020-06-25 at 23 04 04

**Comment by kvviecien** [comment]
@gtandersen I thought putting Send as last would be better since it's best accessible when close to the edge of the screen, so maybe adding a little more space would be better if you think the buttons are too close.

**Comment by gtandersen** [comment]
@kvviecien Alright, let's try that. If it doesn't help we can always make it a bit larger than the other buttons. ;)

**Pull request merged**